src/openvslam_ros.cc

```diff
@@ -45,6 +46,21 @@ void system::publish_pose() {
     pose_msg.pose.pose.position.y = trans(1);
     pose_msg.pose.pose.position.z = trans(2);
     pose_pub_->publish(pose_msg);
+
+    // Create transform message between map->camera_link
+    geometry_msgs::msg::TransformStamped map_trans;
+    map_trans.header.stamp =  node_->now();
+    map_trans.header.frame_id = "map";
+    map_trans.child_frame_id = "camera_link";
+    map_trans.transform.rotation.x = quat.x();
+    map_trans.transform.rotation.y = quat.y();
+    map_trans.transform.rotation.z = quat.z();
+    map_trans.transform.rotation.w = quat.w();
+    map_trans.transform.translation.x = trans(0);
+    map_trans.transform.translation.y = trans(1);
+    map_trans.transform.translation.z = trans(2);
+    map_broadcaster_->sendTransform(map_trans);
```

Comment:
This must be the map->odom transformation

Reply:
You cannot skip like this, publishing the pose to a topic is OK, but you must provide only the map->odom transform or you're introducing cycles in the TF tree which will be rejected or cause issues